src/csharp/Microsoft.Spark/Sql/DataFrame.cs

```diff
@@ -218,6 +218,13 @@ public void Explain(string mode)
         public DataFrameStatFunctions Stat() =>
             new DataFrameStatFunctions((JvmObjectReference)_jvmObject.Invoke("stat"));
 
+        /// <summary>
+        /// Returns the content of the DataFrame as a DataFrame of JSON strings.
+        /// </summary>
+        /// <returns>DataFrame object with JSON strings.</returns>
+        public DataFrame ToJson() =>
```

Comment:
Both scala and python have `JSON` captialized.  Should we do the same ?

Reply:
We can. I was just trying to maintain consistency with respect to the other functions we have such as `FromJson` but since they have similar casing in Scala and Python, let's just go with that. Changed.